packages/medusa/src/loaders/helpers/register-workflows.ts

```diff
@@ -16,6 +16,11 @@ export async function registerProjectWorkflows({
   configModule,
 }) {
   const [resolved] =
-    getResolvedPlugins(rootDirectory, configModule, "dist", true) || []
+    getResolvedPlugins(
```

Comment:
Is this the only place where we discover resources of an app. REMEMBER, WE DON'T WANT TO TOUCH PACKAGES HERE